syft/pkg/cataloger/java/test-fixtures/pom/colon-delimited-with-equals.pom.properties

```diff
@@ -2,4 +2,4 @@
 #Tue Jul 07 18:59:56 GMT 2020
 groupId:org.anchore
 artifactId: example-java=app-maven
-version:   0.1.0=something
+version:   0.1.0=something
```

Comment:
Nit / future: It'd be nice to find easy ways to prevent these newline removals from adding noise to the diff. (For myself, I just check a box in my editor)

Reply:
I'd vote adding precommit hooks over synchronizing IDE settings (something I've been wanting to add to the repo anyway)

Reply:
Either/both is good w/ me 👍